arch/unitroot/unitroot.py

```diff
@@ -505,6 +507,16 @@ def trend(self, value):
             self._reset()
             self._trend = value
 
+    @property
```

Comment:
These "convenience" functions aren't needed IMO. As long as the OLS estimation result is available then it is used to use

adf.regression.pvalue or adf.regression.summary()